### PR TITLE
Feat: Add ktlint

### PR DESCRIPTION
**Pull request by thesohelshaikh**
- Add ktlint using plugin
- Apply ktlintFormat to the project
- Add ktlint to build workflow(ci pipeline)

Fixes: #10 and #23

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -4,18 +4,33 @@ buildscript {
         compose_compiler_version = '1.3.0-rc01'
         compose_version = '1.2.0'
         hilt_version = '2.42'
+        ktlint_version = '10.3.0'
```
Comment (waseefakhtar) [inline build.gradle:7]: Nice that you follow the convention! 👌

File: .github/workflows/android.yml [hunk 2]
```diff
@@ -34,10 +34,13 @@ jobs:
       run: chmod +x gradlew
 
     - name: Build with Gradle
-      run: ./gradlew build
+      run: ./gradlew assemble
```
Comment (waseefakhtar) [inline .github/workflows/android.yml:37]: Why did we change this?

**Review by waseefakhtar — COMMENTED** [review]
Looks good! I just have a minor comment regarding something that looks out of scope of this PR.

File: .github/workflows/android.yml (hunk 2 above)
Comment (thesohelshaikh) [reply .github/workflows/android.yml:37]: `build` depends on `check`, which means `build` already runs tests. This is not the behaviour we need as we run unit tests separately. Consequently, if the `tests` fail then the `build` also fails.

Reference: https://stackoverflow.com/a/61878404

Comment (waseefakhtar) [reply .github/workflows/android.yml:37]: Sounds good! Could we create another PR for it and let this only focus on ktlint?

Comment (thesohelshaikh) [reply .github/workflows/android.yml:37]: Umm, sure. Would it be okay if I keep this as a separate commit?

Comment (waseefakhtar) [reply .github/workflows/android.yml:37]: Yes :)

Comment (thesohelshaikh) [reply .github/workflows/android.yml]: Fixed in latest PR. Will create separate PR for this

**Review by thesohelshaikh — COMMENTED** [review]
LGTM 👍

**Review by waseefakhtar — APPROVED** [review]
Thanks @thesohelshaikh!

**Pull request merged**